network-ntt/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! all the network related actions and processes
+//!
+//! This module only provides and handle the different connections
+//! and act as message passing between the other modules (blockchain,
+//! transactions...);
+
+//extern crate cardano;
+//#[macro_use]
+//extern crate futures;
+//extern crate tokio;
+extern crate protocol_tokio as tokio_protocol;
```

Comment:
Just use `protocol` for brevity?

Reply:
Also, `extern crate` is unnecessary in Rust 2018.

Reply:
`protocol` clashes with `protocol` crate.

Reply:
That one is not imported, though? I don't think swapping words in the name brings more clarity to the code.

Reply:
Yeah that is fixed in the latest versions